src/domain/hanke/edit/Form.tsx

```diff
@@ -99,7 +106,11 @@ const HankeForm: React.FC<Props> = ({
         <div className="hankeForm__formWpr">
           <StateIndicator formPage={formPage} />
           <div className="hankeForm__formWprRight">
-            <form name="hanke" onSubmit={handleSubmit(onSubmit)}>
+            <form
+              name="hanke"
+              // eslint-disable-next-line no-console
+              onSubmit={() => console.error('This should not happen.')}
```

Comment:
Voisitko kommentoida tarkemmin miksi näin ei tulisi tapahtua?

Reply:
Otan mieluummin tämän rivin pois, oli hölmö ajatus lisätä tämä logitus.